### ISSUE TITLE
Gradient Descent with Early Stopping is Provably Robust to Label Noise for Overparameterized Neural Networks

### ISSUE THREAD

**Issue by nocotan** [CompML/stochastic-optimization-survey#17]
## 一言でいうと
停止規則としてearly stoppingを採用したoverparameterized DNNがラベルノイズにロバストであることを示した．

### 論文リンク
AISTATS2020
http://proceedings.mlr.press/v108/li20j/li20j.pdf

### 著者/所属機関
Mingchen Li (University of California), Mahdi Soltanolkotabi (University of Southern California), Samet Oymak (University of California)

### 投稿日付(yyyy/MM/dd)
2019/03/27

## 概要

以下の図のように，overparametrized DNNは
1. 初期のイテレーションではラベルノイズを無視するように振る舞い，
2. 後半のイテレーションでは過学習し始める．

<img width="840" alt="Screen Shot 2021-01-05 at 10 10 31" src="https://user-images.githubusercontent.com/10952293/103595282-640be580-4f3e-11eb-95c9-c9d76559f4e7.png">

## 手法

<img width="416" alt="Screen Shot 2021-01-05 at 10 10 54" src="https://user-images.githubusercontent.com/10952293/103595292-679f6c80-4f3e-11eb-853c-457f8280a051.png">

## 結果

<img width="847" alt="Screen Shot 2021-01-05 at 10 11 13" src="https://user-images.githubusercontent.com/10952293/103595296-6a9a5d00-4f3e-11eb-9733-80e136925ad5.png">

<img width="837" alt="Screen Shot 2021-01-05 at 10 11 22" src="https://user-images.githubusercontent.com/10952293/103595306-6d954d80-4f3e-11eb-8cb1-362e9afb65c8.png">

## コメント

近年のDeep Double Descent現象と矛盾する気もする．
https://arxiv.org/abs/1912.02292
